ecs-cli/modules/cli/compose/entity/service/service_test.go

```diff
@@ -485,6 +485,64 @@ func TestCreateWithServiceDiscoveryWithContainerNameAndPort(t *testing.T) {
 	)
 }
 
+func TestCreateWithSchedulingStrategyWithDaemon(t *testing.T) {
+	schedulingStrategy := ecs.SchedulingStrategyDaemon
+
+	flagSet := flag.NewFlagSet("ecs-cli-up", 0)
+	flagSet.String(flags.SchedulingStrategyFlag, schedulingStrategy, "")
+
+	createServiceTest(
+		t,
+		flagSet,
+		&config.CommandConfig{},
+		&utils.ECSParams{},
+		func(input *ecs.CreateServiceInput) {
+			actual := input
+			assert.NotNil(t, actual.SchedulingStrategy, "SchedulingStrategy should not be nil")
+			assert.Equal(t, schedulingStrategy, aws.StringValue(actual.SchedulingStrategy), "SchedulingStrategy should match")
+			assert.Nil(t, actual.DesiredCount, "DesiredCount should be nil")
+		},
+	)
+}
+
+func TestCreateWithSchedulingStrategyWithReplica(t *testing.T) {
+	schedulingStrategy := "REPLICA"
```

Comment:
nit: Use `ecs.SchedulingStrategyReplica` here

Reply:
https://github.com/aws/amazon-ecs-cli/pull/685/commits/6dace11db3e0dc1a06c468c4716069a8c839bf57 fixes this